port/test/u_port_test.c

```diff
@@ -75,8 +75,14 @@
 
 #include "u_test_util_resource_check.h"
 
-#ifdef CONFIG_IRQ_OFFLOAD
-# include <irq_offload.h> // To test semaphore from ISR in zephyr
+#include <version.h>
```

Comment:
This file is slightly different to the others in that it is compiled for all platforms that `ubxlib` supports, so the `#include <version.h>` will need to either come inside the `#ifdef CONFIG_IRQ_OFFLOAD` or, if you prefer, you can put the lot under `#ifdef __ZEPHYR__`.